jc/parsers/efibootmgr.py

```diff
@@ -0,0 +1,99 @@
+"""jc - JSON Convert `efibootmgr` command output parser
+
+Only `efibootmgr -v` option is supported with the default output.
+
+Usage (cli):
+
+    $ sudo efibootmgr | jc --efibootmgr
+    $ sudo efibootmgr -v | jc --efibootmgr
+
+or
+
+    $ sudo jc efibootmgr
+
+Usage (module):
+
+    import jc
+    result = jc.parse('efibootmgr', efibootmgr_command_output)
+
+Schema:
+
+    [
+      "BootCurrent": "0000",
+      "Timeout": "0 seconds",
+      "BootOrder": ["0000","0001"],
+      "BootOptions": [
+        {
+          "BootOptionReference": "Boot0000",
+          "DisplayName": "opensuse-secureboot",
+          "UefiDevicePath": "HD(1,GPT,e45dbd72-ba93-4810-9330-2c1c493ff08b,0x800,0x31800)/File(\EFI\opensuse\shim.efi)",
+          "BootOptionEnabled": true
+        }
+      ]
+    ]
```

Comment:
It looks like you may need to wrap your schema in an object:
```
[
  {
    "BootCurrent": "0000",
    "Timeout": "0 seconds",
    "BootOrder": ["0000","0001"],
    "BootOptions": [
      {
        "BootOptionReference": "Boot0000",
        "DisplayName": "opensuse-secureboot",
        "UefiDevicePath": "HD(1,GPT,e45dbd72-ba93-4810-9330-2c1c493ff08b,0x800,0x31800)/File(\EFI\opensuse\shim.efi)",
        "BootOptionEnabled": true
      }
    ]
  }
]
```

Reply:
The schema was updated.